### DISCUSSION TITLE
Welcome to seestar_run Discussions!

### DISCUSSION THREAD

**Discussion by smart-underworld** [smart-underworld/seestar_run#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Brnikat** [comment]
Hi, I'm Paul Leyland and my main observatory is in La Palma where my major research interests are photometry of asteroids, variable stars and exoplanetary transits. I bought a Seestar S50 a week ago so that I can do at least a little astronomy in the UK.

Seestar_run looks to be very useful and I've downloaded it but the sky hasn't let me use it yet.

I note that RA and Dec must be given in Jnow but virtually all catalogues use J2000. Precessing between the two is relatively straightforward and I suggest that a CLI option be given which allows either of them.  I further suggest that J2000 becomes the default.

Perhaps I may hack the code some time and send it on for consideration.

**Comment by smart-underworld** [comment]
Seestar_alp takes J2000 as well. There's a checkbox to indicate the
reference used.

**Comment by smart-underworld** [comment]
Btw, use seestar_alp. It's the next gen version of seetar_run, and is
being  actively developed

**Reply by Brnikat** [reply]
Thanks.  Downloaded the Linux version of seestar_alp and got this result on a Ubuntu 24.04LTS box:
```

pcl@thoth:~/Downloads/seestar_alp$ ./seestar_alp
Exception in thread FrontMainThread:
Traceback (most recent call last):
  File "threading.py", line 1075, in _bootstrap_inner
  File "threading.py", line 1012, in run
  File "root_app.py", line 58, in runner
  File "app.py", line 2736, in start
  File "wsgiref/simple_server.py", line 154, in make_server
  File "socketserver.py", line 457, in __init__
  File "wsgiref/simple_server.py", line 50, in server_bind
  File "http/server.py", line 136, in server_bind
  File "socketserver.py", line 473, in server_bind
OSError: [Errno 98] Address already in use
Startup Complete
pcl@thoth:~/Downloads/seestar_alp$ 

```

I have no idea what else may be using the port. Too late now and I am too tired to investigate further. Tomorrow, perhaps.

**Comment by Brnikat** [comment]
Building seestar_alp from a git clone fails on the same 24.04LTS system as follows:

```
pcl@thoth:~/Astro/Code$ git clone https://github.com/smart-underworld/seestar_alp.git
Cloning into 'seestar_alp'...
remote: Enumerating objects: 4331, done.
remote: Counting objects: 100% (174/174), done.
remote: Compressing objects: 100% (115/115), done.
remote: Total 4331 (delta 79), reused 138 (delta 59), pack-reused 4157 (from 1)
Receiving objects: 100% (4331/4331), 94.67 MiB | 7.90 MiB/s, done.
Resolving deltas: 100% (2706/2706), done.
pcl@thoth:~/Astro/Code$ cd se*p
pcl@thoth:~/Astro/Code/seestar_alp$ python install -r requirements.txt
python: can't open file '/home/pcl/Astro/Code/seestar_alp/install': [Errno 2] No such file or directory
pcl@thoth:~/Astro/Code/seestar_alp$ 

```

I don't seem to be doing very well, do I?
